pkg/kernel/patches-5.10.x/0014-net-fec-Silence-M5272-build-warnings.patch

```diff
@@ -0,0 +1,72 @@
+From 0eb93b443100b98e7e2fe0e1c8c32673fdb6e952 Mon Sep 17 00:00:00 2001
```

Comment:
There is already the following patch starting by 0014: 0014-Add-support-for-LEDs-on-Operational-Services-EM320-d.patch

Reply:
Thanks! Fixed